ignored-tests.json

```diff
@@ -124,6 +124,7 @@
   "Launcher specs Puppeteer Puppeteer.connect should support targetFilter option in puppeteer.launch",
   "Launcher specs Puppeteer Puppeteer.connect should support targetFilter option",
   "Launcher specs Puppeteer Puppeteer.launch should launch Chrome properly with --no-startup-window and waitForInitialPage=false",
+  "Launcher specs Puppeteer Puppeteer.launch should work with no default arguments",
```

Comment:
The test generally works but might fail on CI as it would enable user-facing UI in Chrome.